src/brain/gocdSlackFeeds/deployFeed.test.ts

```diff
@@ -516,7 +516,7 @@ describe('DeployFeed', () => {
   });
 
   it('post message with commits in deploy link for getsentry', async () => {
-    const octokit = await getClient(ClientType.App, GETSENTRY_ORG);
+    const octokit = await getClient(ClientType.App, 'Enterprise');
```

Comment:
We are [mocking](https://github.com/getsentry/eng-pipes/blob/0ca5a8ad0b4e37adba26dae1025171d20b635bb1/src/api/github/__mocks__/octokitWithRetries.ts) the replies. 🐭

Reply:
Oooh ... this one wants to be `GETSENTRY_ORG` though, good catch!

We end up testing both `getClient` cases pretty thoroughly since both are called a ton throughout the test suite.